docs/4.0/docs/cloud/Intro.md

```diff
@@ -1,38 +1,38 @@
 # sealos Cloud introduction
 
-sealos Cloud is a sealos cluster operated and maintained by labring team to provide public cloud services.
-Users can directly use [sealos Cloud](https://cloud.sealos.io),
-Users can also run sealos in their own private environment to have the same capabilities as sealos Cloud.
+sealos Cloud is the sealos clusters operated and maintained by labring team which provide public cloud services.
+Users can directly use [sealos Cloud](https://cloud.sealos.io)
+Users can also run sealos in their own environment to have the same capabilities as sealos Cloud.
 
-sealos Cloud is an ALL IN cloud-native public cloud service, 
+sealos Cloud is a public cloud service purely based on cloud-native technology. It provides common service capabilities in public clouds, 
 such as database services, function computing services, and object storage services.
 
 The biggest difference from public clouds such as AWS is that the technical architecture is completely implemented using 
 cloud-native (built around the technical ecosystem of kubernetes) architecture and 100% opensource.
 
 ## why sealos Cloud
 
-* The product is simple and clean, if you have been tortured by the dazzling advertising discount information of other Cloud provider.
-* Completely open source, which means unlimited and free to customize.
-* The public cloud and private cloud is exactly the same. The public cloud and private cloud of sealos have the same set of codes. 
-* Runs everywhere, compatible with various other cloud infrastructures, compatible with bare metal, supports various architectures and mainstream linux operating systems.
-* The price is cheap, and privatization can help enterprises save 80% of infrastructure costs, and direct use of public clouds can save 10% to 40% of costs.
+* The product is simple and clean. You won't be tortured by the dazzling advertisements published by other Cloud providers.
```

Comment:
Here, we means "dazzling advertising discount information", but not "azzling advertisements".

Reply:
Done